### ISSUE TITLE
False positive when a list of animations is used in no-unknown-animations

### ISSUE THREAD

**Issue by thykka** [stylelint/stylelint#1902]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

If `animation-name` contains more than one identifier, the subsequent identifiers are incorrectly detected as `no-unknown-animations`, regardless of whether the animations are actually defined. 

> Which rule, if any, is this issue related to?

`no-unknown-animations`

> What CSS is needed to reproduce this issue?

```css
.myElement {
  animation-name: doThis, doThat;
}

@keyframes doThis {
  0% { opacity: 0; }
  100% { opacity: 1; }
}

@keyframes doThat {
  0% { opacity: 1; }
  100% { opacity: 0; }
}
```

> What stylelint configuration is needed to reproduce this issue?

```json
{
  "rules": {
    "no-unknown-animations": true,
  }
}
```

> Which version of stylelint are you using?

`6.0.3`

> How are you running stylelint: CLI, PostCSS plugin, Node API?

PostCSS/CSSNext via Gulp

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

no.

> What did you expect to happen?

No warnings to be flagged.

> What actually happened (e.g. what warnings or errors you are getting)?

The following warning was flagged:

```shell
test.css
2:19  ✖  Unexpected unknown animation name "doThis, doThat"   no-unknown-animations
```

**Comment by ntwb** [comment]
I'm pretty sure this was fixed in stylelint 7.2 @thykka, could you please update to the latest version

https://github.com/stylelint/stylelint/releases/tag/7.2.0
> Fixed: no-unknown-animations and unit-blacklist now handle numbers without leading zeros.

**Comment by thykka** [comment]
Indeed, the problem does not persist after upgrading to 7.2.0. Thanks for pointing this out @ntwb!

**Comment by thykka** [comment]
Oops, I might've been too fast with this.. Some deprecated stylelint-rules were throwing an error, hiding the actual linting errors, which still appear on 7.2.0;

```console
src/css/test.css
 2:19  ✖  Unexpected unknown animation name "doThis, doThat"   no-unknown-animations
```
```console
$ npm list stylelint
myproject@1.0.0 /Users/thykka/wwwroot/myProject
└─┬ gulp-stylelint@3.2.0
  └── stylelint@7.2.0 
```

**Comment by davidtheclark** [comment]
Yep, looks like the rule does not account for this case yet (no tests involved it).

@thykka, if you'd like to contribute to stylelint, go for it!

**Comment by jeddy3** [comment]
@thykka You'll find information on fixing bugs [in the developer guide](https://github.com/stylelint/stylelint/blob/master/docs/developer-guide/rules.md#fixing-bugs). Thanks for the nice and clear original issue btw :) I hadn't realised, until now, it's possible to have [lists of animations](https://developer.mozilla.org/en/docs/Web/CSS/animation-name)...
